azurelinuxagent/ga/periodic_operation.py

```diff
@@ -33,9 +33,8 @@ class PeriodicOperation(object):
     # To prevent flooding the log with error messages we report failures at most every hour
     _LOG_WARNING_PERIOD = datetime.timedelta(minutes=60)
 
-    def __init__(self, name, operation, period):
-        self._name = name
-        self._operation = operation
+    def __init__(self, period):
```

Comment:
Now that all the code is refactored to use subclasses of PeriodicOperation, there is no need of a string to identify the operation (we can use the type) nor the need to pass the function to execute as parameter (this is now done using inheritance)